a3m/client/clientScripts/change_object_names.py

```diff
@@ -151,40 +151,40 @@ def apply_file_updates(self):
             event.agents.add(*event_agents)
 
         if len(self.files_index) > 0:
-            logger.debug("Sanitized batch of %s files", len(self.files_index))
+            logger.debug("Changed batch of %s files", len(self.files_index))
 
             self.files_index = {}
         else:
-            logger.debug("No file sanitization required.")
+            logger.debug("No filename change required.")
 
     def apply_dir_updates(self):
         """
         Run a single batch of Directory updates.
         """
         if self.directory_queryset is None:
-            logger.debug("No directory sanitization required.")
+            logger.debug("No directory name change required.")
             return
 
         # We pass through _all_ objects here, as they may not be normalized in
         # the db :(
         for dir_obj in self.directory_queryset.iterator():
             old_location = unicodedata.normalize("NFC", dir_obj.currentlocation)
             try:
-                sanitized_location = self.dirs_index[old_location]
+                changed_location = self.dirs_index[old_location]
             except KeyError:
                 continue
 
-            dir_obj.currentlocation = sanitized_location
+            dir_obj.currentlocation = changed_location
             dir_obj.save()
 
-            # TODO: Dir sanitizations don't generate events?
+            # TODO: Dir name changes don't generate events?
             # Is seems like they should.
 
         if len(self.dirs_index) > 0:
-            logger.debug("Sanitized batch of %s directories", len(self.dirs_index))
+            logger.debug("Changed batch of %s directories", len(self.dirs_index))
```

Comment:
I think this should specify that the change is a name change, e.g. "Changed batch of %s directory names"